README.md

```diff
@@ -94,6 +94,12 @@ yarn install
 yarn dev
 ```
 
+To use react dev tools, you can use [electron-react-devtools](https://github.com/firejune/electron-react-devtools), which should already be installed. When you open devtools, type the following into the console:
+```javascript
+require('electron-react-devtools').install()
```

Comment:
Should we be doing this from the main.ts script when we are in dev mode?

Reply:
Let me look into this. :)

Reply:
Did you mean the index file? I've tried there but it must be called from the renderer process, so investigating now if there's somewhere else I can call install.

Reply:
Ahh - it's not even working sending a message as we did earlier :P

Reply:
Let's get it in and see for now.